create3_control/controllers/factory.py

```diff
@@ -18,10 +34,11 @@ def construct_fb_linearization_controller(ros2_node):
     # Construct controller
     gain = ros2_node.get_parameter(gain_param_name).get_parameter_value().double_value
     length = ros2_node.get_parameter(lenght_param_name).get_parameter_value().double_value
-    controller = FBLinearizationController(gain, length)
+    controller = FBLinCtrl(gain, length)
```

Comment:
abbreviations like this are very confusing

Reply:
ok